### DISCUSSION TITLE
ERROR "Cannot find file" _projects, when I delete all projects from folder

### DISCUSSION THREAD

**Discussion by AndreaMoschetto** [alshedivat/al-folio#2251] [Q&A] [answered]
If I delete all projects from _projects folder the link checker says:
`[ERR] file:///home/runner/work/AndreaMoschetto.github.io/AndreaMoschetto.github.io/_projects | Failed: Cannot find file`

I think it's because if you delete all files inside a folder, that empty folder disappears from GitHub and than the link checker doesn't
see it. How can I leave the "_projects" folder empty without raising errors?

**Comment by george-gca** [comment, marked as answer]
Probably the error is due to the projects collection defined in `_config.yml`. Remove the lines that reference it and it should be fine.
